### PR TITLE
Fix UI

### PR DESCRIPTION
**Pull request by osamaasifoak**
This pull request only includes user interface fixes that were pointed out by @scouech. The changes were focused mainly on the Home, Meditation, and Folder screens. I am currently still working on resolving other issues.

### CODE REVIEW

**Comment by michaelcspeed** [comment]
@osamaasifoak I'm not gonna be able to review this until probably Tuesday, would you be able to give it a self review before then please? Just a quick glance I can see some comments that probably need removing, stuff like that. Thanks!

**Comment by osamaasifoak** [comment]
Yeah sure ;)

File: lib/services/notifications/notifications_service.dart [hunk 1]
```diff
@@ -59,10 +59,7 @@ Future<void> initialiazeLocalNotification() async {
   );
 }
 
-void onSelect(NotificationResponse? data) async {
-  // TODO
-  print(data);
-}
+void onSelect(NotificationResponse? data) async {}
```
Comment (michaelcspeed) [inline lib/services/notifications/notifications_service.dart]: What about these warnings @osamaasifoak ?

File: lib/views/home/widgets/bottom_sheet/stats/stats_bottom_sheet_widget.dart [hunk 2]
```diff
@@ -46,52 +41,48 @@ class StatsBottomSheetWidget extends ConsumerWidget {
...
+        RepaintBoundary(
+          key: key,
+          child: Container(
+            color: ColorConstants.onyx,
+            child: Column(
+              children: stats.all.map((e) {
+                return RowItemWidget(
+                  iconCodePoint: e.icon,
+                  iconColor: e.color,
+                  trailingIconSize: 20,
```
Comment (michaelcspeed) [inline lib/views/home/widgets/bottom_sheet/stats/stats_bottom_sheet_widget.dart:66]: should this be in a constant?

**Review by michaelcspeed — APPROVED** [review]
Could you check the other warnings about repeated functions etc before merging? Thanks

File: lib/services/notifications/notifications_service.dart (hunk 1 above)
Comment (osamaasifoak) [reply lib/services/notifications/notifications_service.dart]: Removed

File: lib/views/home/widgets/bottom_sheet/stats/stats_bottom_sheet_widget.dart (hunk 2 above)
Comment (osamaasifoak) [reply lib/views/home/widgets/bottom_sheet/stats/stats_bottom_sheet_widget.dart:66]: Created a constant for trailingIconSize.

**Pull request merged**